src/App.tsx

```diff
@@ -2,15 +2,22 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+  const addnewMovies = (movie: Movie) => {
```

Comment:
```suggestion
  const addMovie = (movie: Movie) => {
```
you're adding 1 movie here
FYI